<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vipDialog</class>
 <widget class="QDialog" name="vipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1_2">
    <attribute name="title">
     <string>查询VIP</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>VIP号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_vip_id"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_vip_submit">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>421</width>
       <height>175</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_12">
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="search_vip_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>积分:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="search_vip_score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2_2">
    <attribute name="title">
     <string>添加VIP</string>
    </attribute>
    <widget class="QPushButton" name="addPerson">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>401</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_11">
      <item row="2" column="0">
       <widget class="QLabel" name="Label_61">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="create_vip_name"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_65">
        <property name="text">
         <string>手机号码</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="create_vip_phone"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="create_vip_id"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label_60">
        <property name="text">
         <string>VIP号 </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>删除VIP</string>
    </attribute>
    <widget class="QPushButton" name="submit_delete">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>330</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认删除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>401</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_54">
        <property name="text">
         <string>员工号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="delete_vip_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_55">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="request_info">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="delete_vip_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_56">
        <property name="text">
         <string>积分</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="delete_vip_score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>VIP积分兑换</string>
    </attribute>
    <widget class="QPushButton" name="update_clear">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit_update">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="Label_18">
        <property name="text">
         <string>VIP号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="update_vip_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>兑换积分</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="update_vip_score"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
